CUSTOM.md

```diff
@@ -370,3 +370,26 @@ This section is meant for users who are familiar with both the tts mod and progr
         - 4: Slow Phase
         - 5: Time Passes
       - `turn`: **number** - is the current turn number
+- Trigger to modify the cost of played cards
+  - Create object and tag with "Modify Cost"
+  - `modifyCost(params)`
+    - `params`: **table** - contains data about the player and the cards played
+      - `color`: **string** - is the color of the player whose card cost is being calculated
+      - `costs`: **table** - maps card GUIDs (as keys) to costs (as values)
```

Comment:
since this table doesn't end up getting saved somewhere, i think it may be better to just pass the object directly? not really familiar with TTS performance though so perhaps it's better to just pass a guid and then perform object lookup again rather than pass the object? just feels a bit off to convert an object into a guid and then back into an object

Reply:
Huh. I had assumed, based on what I know of hash tables in other languages, that using an object reference as the key in a table wouldn't work, and that it'd need a simple data type (e.g. a string). But I've just tested it, and it seems like using object references as they keys in a table *does* work.

I also don't know about TTS performance to say whether using an object reference as a table key has performance implications worse than doing the object lookup again. I can try doing some benchmarking.